spec/unit/intercom/request_spec.rb

```diff
@@ -1,5 +1,9 @@
 require 'spec_helper'
 require 'ostruct'
+require 'webmock'
+include WebMock::API
+
+WebMock.enable!
```

Comment:
Can we shift this up into to the spec_helper file? webmock can come in handy for other tests in the future.

Reply:
Yeah, I was initially unsure whether to put it in spec_helper file in case it impacted other tests but will do it now. It should not impact other tests